### PR TITLE
Routing

### LINKED ISSUE #82: Reactivate authorized pages
Last time, the code was commented to bypass the login page, and we can just access the admin pages without being authenticated. Let's remove the commented code and have the pages to be accessible by authorized user!

### PR DESCRIPTION
**Pull request by theodorusandi**
closes #82 #97
New folder structures + cookie feature

### CODE REVIEW

File: src/components/DropDownMenuItem/DropDownMenuItem.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const StyledDropDownMenuItem = styled.div`
+  position: relative;
+  width: 100%;
+  background-color: ${({ theme }) => theme.background.primary};
+`;
+
+export const StlyedDropDownMenuItemHeader = styled.div`
```
Comment (RDiMag) [inline src/components/DropDownMenuItem/DropDownMenuItem.styled.tsx:9]: I think this is a typo in line 9: "Stlyed".

File: src/components/DropDownMenuItem/DropDownMenuItem.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  position: relative;
+  width: 100%;
+  padding: 0.5em;
+  border-bottom: 1px solid ${({ theme }) => theme.background.white};
+  display: flex;
+  gap: 0.5em;
+`;
+
+export const StlyedDropDownMenuItemBody = styled.div`
```
Comment (RDiMag) [inline src/components/DropDownMenuItem/DropDownMenuItem.styled.tsx:18]: Same typo here: "Stlyed".

File: src/components/DropDownMenuItem/DropDownMenuItem.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  StlyedDropDownMenuItemBody,
+  StlyedDropDownMenuItemHeader,
+  StyledDropDownMenuItem,
+} from './DropDownMenuItem.styled';
```
Comment (RDiMag) [inline src/components/DropDownMenuItem/DropDownMenuItem.tsx:6]: There are some typos in the import for lines 3-5.

**Review by RDiMag — APPROVED** [review]
There are some minor typos that need a fix, otherwise it is approved!

**Review by jpatrici — APPROVED** [review]

**Pull request merged**